### ISSUE TITLE
github preview output not responsive.

### ISSUE THREAD

**Issue by R4wd0g** [facelessuser/MarkdownPreview#180]
Hello folks. Here i cannot manage to make the github markdown browser window to be responsible when changing the window size of the browser.
It's weird cause this behavior doens't occur on others parsers like markdown, or gitlab.
thanks

<img width="387" alt="img" src="https://github.com/facelessuser/MarkdownPreview/assets/29448208/2b462938-b51f-4ba4-b54b-b94cc7f78591">

**Comment by R4wd0g** [comment]
thanks for answering.
Can u see that the text are cropped? The text formatting does't follow the window resize.
How would you name this phenomenon?

**Comment by R4wd0g** [comment]
> I don't think I understand what you are describing.

thanks for answering.
Can u see that the text are cropped? The text formatting does't follow the window resize.
How would you name this phenomenon?

**Comment by facelessuser** [comment]
It probably has a minimum width.

**Comment by R4wd0g** [comment]
> It probably has a minimum width.

it's weird, because on parser "markdown" or "gitlab", this behavior doesn'r occur.
Do happens the same at your side?

**Comment by facelessuser** [comment]
Somehow we have a discrepancy where the others use a max size of \<somesize> and GitHub uses a size of \<somesize>.  One allows shrinking, the other does not.

**Comment by R4wd0g** [comment]
> Somehow we have a discrepancy where the others use a max size of <somesize> and GitHub uses a size of <somesize>. One allows shrinking, the other does not.

But, on GitHub website, the text formatting follows the window size. You can compare by yourself at readme page:
https://github.com/facelessuser/MarkdownPreview/


Sorry if it isn't a bug. But some limitation of the plugin.

**Comment by facelessuser** [comment]
No, i just mentioned there is a discrepancy in the CSS we provide for GitHub and the CSS we provide for the others. That means we need to update our GitHub CSS.

**Comment by facelessuser** [comment]
Fix is here: https://github.com/facelessuser/MarkdownPreview/commit/c94a1e0465188b4b526d95adeef773b88f72b928

**Comment by R4wd0g** [comment]
> Fix is here: [c94a1e0](https://github.com/facelessuser/MarkdownPreview/commit/c94a1e0465188b4b526d95adeef773b88f72b928)

Thanks @facelessuser , you approach has fixed the issue successfully. Kindly, i made a pull request of a minimal modification to make the content fit better.
bless you
